src/components/ThemeProvider.js

```diff
@@ -16,7 +16,61 @@ const theme = createMuiTheme({
                 fontFamily,
                 fontSize: '.7rem',
             }
-        }
+        },
+        MuiCard: {
+            root: {
+                background: 'rgb(174, 196, 244, 0.19)',
+                color: '#FFFFFF'
+            }
+        },
+        MuiInput: {
+            root: {
+                color: '#FFFFFF'
+            }
+        },
+        MuiFormLabel: {
+            root: {
+                color: '#AEC4F4'
+            }
+        },
+        MuiSvgIcon: {
+            root: {
+                color: '#FFFFFF'
+            }
+        },
+        MuiIcon: {
+            root: {
+                color: '#FFFFFF'
+            }
+        },
+        MuiSelect: {
+            root: {
+                color: '#FFFFFF'
```

Comment:
Does the theme not support setting the global root color? :(

Reply:
Additionally there might be a way to set the default label colour globally, it might be textSecondary colour or something https://material-ui.com/api/typography/

Reply:
idk if there's a way to set global root color, i think i set the default label colors tho here: 
![image](https://user-images.githubusercontent.com/45786074/85217547-0c61b000-b347-11ea-96fe-8ea80a46bfd1.png)